### PR TITLE
Add ruby code sample for test (#40)

### LINKED ISSUE #40: Add ruby code sample for tests
Add ruby code samples for tests on analyzer and scanner

### PR DESCRIPTION
**Pull request by ChandanChainani**
Fixes #40

### CODE REVIEW

**Review by JoaoDanielRufino — CHANGES_REQUESTED** [review]

**Comment by ChandanChainani** [comment]
@JoaoDanielRufino

Few cases which are currently not working for ruby code:
- line with code and comment together
 ```
 puts "Welcome to Ruby!"     # prints the welcome message
 ```
- other form of comment that is allowed in ruby
  ```
  =begin
  This is a comment line
  it explains that the next line of code displays
  a welcome message
  =end
  ```

**Comment by JoaoDanielRufino** [comment]
> @JoaoDanielRufino
> 
> Few cases which are currently not working for ruby code:
> 
> * line with code and comment together
> 
> ```
> puts "Welcome to Ruby!"     # prints the welcome message
> ```
> 
> * Other form of comment that is allowed in ruby
>   ```
>   =begin
>   This is a comment line
>   it explains that the next line of code displays
>   a welcome message
>   =end
>   ```
@ChandanChainani 

The line with code and comment together should not be counted as a comment line, so the code is working properly.

The block comment is supported by gcloc, could you add it on the code ? I think you didn't understand my review when I asked for it

File: test/fixtures/code_samples/main.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Given an array of 0s and 1s in random order.
+# Segregate 0s on left side and 1s on right side of the array
```
Comment (JoaoDanielRufino) [inline test/fixtures/code_samples/main.rb:2]: Could you convert this to a block comment ?
```suggestion
=begin
Given an array of 0s and 1s in random order.
Segregate 0s on left side and 1s on right side of the array
=end
```

**Review by JoaoDanielRufino — CHANGES_REQUESTED** [review]

**Review by JoaoDanielRufino — APPROVED** [review]
Thanks for the amazing contribution!

**Pull request merged**